docs/classification_fallback.md

```
@@ -0,0 +1,45 @@
+
+*   **Complaint**
+*   **Opinion/Order**
+*   **Settlement**
+
+All incoming documents from the Civil Rights Litigation Clearinghouse API (CRLCA) or other sources must be assigned to one of these types or gracefully marked as `"Unknown"` if unsupported.
+
+## 2. Order of Precedence
+
+The classification module evaluates a document in the following deterministic order:
+
+### 2.1. Exact Match
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)